### PR TITLE
first version

### PR DESCRIPTION
**Pull request by teemow**
a small python image base on alpine. 

uses `onbuild` to add the python app and install pip packages.

### CODE REVIEW

**Comment by teemow** [comment]
ping @marians @luebken @kordless

File: README.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Small python docker image
+
+Based on alpine.
+
+See https://github.com/gliderlabs/docker-alpine
+
+## Usage
+
+Just derive from the image in `FROM` and the current directory will be added to the image and pip packages will be installed.
```
Comment (marians) [inline README.md]: I think it should say that the current directory will be added as `/app`. An important detail IMHO.

**Comment by marians** [comment]
Nice. The line `FROM giantswarm/python` indicates that this will be on the public registry, right?

**Comment by marians** [comment]
How about naming it `giantswarm/python-alpine`, to make it more clear what it's based on?

**Comment by marians** [comment]
How about adding a `.dockerignore` that contains `.git`?

**Comment by teemow** [comment]
> Nice. The line FROM giantswarm/python indicates that this will be on the public registry, right?

yes

> How about naming it giantswarm/python-alpine, to make it more clear what it's based on?

i don't mind. it will be easier to understand for the uneducated without alpine, but better for the expert :)

> How about adding a .dockerignore that contains .git?

yes. I'll do that

File: README.md (hunk 1 above)
Comment (teemow) [inline README.md]: Done

File: README.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
+# Small python docker image
+
+Based on alpine.
+
+See https://github.com/gliderlabs/docker-alpine
+
+## Usage
+
+Just derive from the image in `FROM` and the current directory will be added to `/app/` in the image and pip packages will be installed.
```
Comment (marians) [inline README.md]: I'd prefer `added as /app/` instead of `added to /app/`.

The current way it's not clear whether mydir might become `/app/mydir`.

**Comment by marians** [comment]
Just a language/clarity thing. Otherwise looks good to me.

**Pull request merged**